utils.js

```diff
@@ -32,18 +32,21 @@ var nestedGet = function(key, obj) {
         if (_.isArray(obj)) {
             dotKey = parseInt(dotKey);
         }
-        var dotValue = obj[dotKey];
+        dotValue = obj[dotKey];
     } catch (ex) {
         return undefined;
     }
 
-    if (remKey.length == 0) return dotValue;
+    if (remKey.length === 0) return dotValue;
     return nestedGet(remKey, dotValue);
-}
+};
 
 // printf-style formatting: format("Hi {0}", "There") => "Hi There"
 var format = function(str, args) {
     if (!args) args = [];
+    if (!_.isArray(args))
```

Comment:
@gregstuderoaf small update --- because this function was doing 
`format("Hi {0}", "There") => "Hi T"`

Reply:
great catch.